nevergrad/optimization/oneshot.py

```diff
@@ -58,18 +74,21 @@ class RandomSearchMaker(base.ParametrizedFamily):
     scale: float or "random"
         scalar for multiplying the suggested point values. If "random", this
         used a randomized pattern for the scale.
+    recommendation_rule: str
+        "average_of_k_best" or "none"; "none" is the default and implies selecting the pessimistic best.        
```

Comment:
"average_of_bests" ("k" is not very informative...) or "pessimistic" ("none" is not informative either :p, and in Python the None variable would be prefered instead, but since there is a clear default here, "pessimistic" is better)